components/badge/badge.css

```diff
@@ -0,0 +1,114 @@
+/* parent selector */
+
+.badge {
+  position: relative;
+}
+
+/* Icon badges */
+
+.icon {
+  font-size: var(--size-12);
+  cursor: pointer;
+}
+
+.unit {
+  position: absolute;
+  border-radius: var(--radius-rounded);
+  min-width: var(--size-12);
+  height: var(--size-12);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: var(--size-3);
+  font-size: var(--size-9);
+  font-weight: var(--weight-medium);
+}
+
+.top-right {
+  top: 0;
+  right: 0;
+  transform: translate(50%, -50%);
+}
+
+.top-left {
+  top: 0;
+  left: 0;
+  transform: translate(-50%, -50%);
+}
+
+.badge-blue {
+  background-color: var(--badge-blue);
+  color: var(--color-white);
+}
+
+.badge-purple {
+  background-color: var(--badge-purple);
+  color: var(--color-white);
+}
+
+.badge-yellow {
+  background-color: var(--badge-yellow);
+  color: var(--color-black);
+}
+
+.badge-red {
+  background-color: var(--badge-red);
+  color: var(--color-white);
+}
+
+/* Avatar badge dot */
+.badge-dot-xl {
+  width: var(--size-10);
+  height: var(--size-10);
+  border-radius: var(--radius-rounded);
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.badge-dot-lg {
+  width: var(--size-9);
+  height: var(--size-9);
+  border-radius: var(--radius-rounded);
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.badge-dot-md {
+  width: var(--size-7);
+  height: var(--size-7);
+  border-radius: var(--radius-rounded);
```

Comment:
these 4 properties are repeating in the above 3 classes.

Reply:
Noted, have created a new CSS class property to store these 4 lines and use them in the respective badges